lib/Twig/Profiler/Dumper/Blackfire.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) 2015 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class Twig_Profiler_Dumper_Blackfire implements Twig_Profiler_DumperInterface
+{
+    public function dump(Twig_Profiler_Profile $profile)
+    {
+        $data = array();
+        $this->dumpChildren('main()', $profile, $data);
+        $this->dumpProfile('main()', $profile, $data);
```

Comment:
I see an inconsistency here: the root profile is dumped after its children, while other profiles are dumped before their children. What is the reason ?